system/Config/Services.php

```diff
@@ -677,7 +677,7 @@ public static function security(App $config = null, bool $getShared = true)
 			return static::getSharedInstance('security', $config);
 		}
 
-		$config = $config ?? config('App');
+		$config = $config ?? config('App') ?? config('Security');
```

Comment:
I think the check for Security config should come first before App.
```suggestion
		$config = $config ?? config('Security')  ?? config('App');
```